pandas/tseries/frequencies.py

```diff
@@ -19,30 +19,19 @@
 from pandas.util._decorators import cache_readonly, deprecate_kwarg
 import pandas.tseries.offsets as offsets
 
-from pandas._libs import lib, tslib
+from pandas._libs import tslib
 from pandas._libs.tslib import Timedelta
+from pandas._libs.tslibs.frequencies import (
+    get_freq_code, _base_and_stride,
+    _INVALID_FREQ_ERROR, opattern,
+    _lite_rule_alias, _dont_uppercase)
 from pandas._libs.tslibs.frequencies import (  # noqa
```

Comment:
make a note that these are 'public'

maybe instead of individually importing the functions from tslibs.frequencies, just

```
from pandas._libs.tslibs import frequencies as fl
``` 
and use 

(to avoid polluting the namespace)

Reply:
Sounds good.